libs/zzz/ui/src/Character/StatDisplay.tsx

```diff
@@ -8,10 +8,12 @@ export function StatDisplay({
   statKey,
   showPercent = false,
   disableIcon = false,
+  showStatName = true,
```

Comment:
Instead of adding an option not to display the name, I think you should just use `StatIcon` component instead. (since you only want the icon)